exchanges/lbank/lbank_wrapper.go

```diff
@@ -0,0 +1,440 @@
+}
+
+// GetWebsocket returns a pointer to the exchange websocket
+func (l *Lbank) GetWebsocket() (*exchange.Websocket, error) {
+	return nil, common.ErrNotYetImplemented
+}
+
+// GetActiveOrders retrieves any orders that are active/open
+func (l *Lbank) GetActiveOrders(getOrdersRequest *exchange.GetOrdersRequest) ([]exchange.OrderDetail, error) {
+	finalResp := make([]exchange.OrderDetail, 0)
+	var resp exchange.OrderDetail
+	tempData, err := l.GetAllOpenOrderID()
+	if err != nil {
+		return finalResp, err
+	}
+
+	for key, val := range tempData {
+		for x := range val {
+			tempResp, err := l.QueryOrder(key, val[x])
+			if err != nil {
+				return finalResp, err
+			}
+			resp.Exchange = l.GetName()
+			resp.CurrencyPair = currency.NewPairFromString(key)
+			if strings.EqualFold(tempResp.Orders[0].Type, "buy") {
+				resp.OrderSide = exchange.BuyOrderSide
+			} else {
+				resp.OrderSide = exchange.SellOrderSide
+			}
+			if tempResp.Orders[0].Status == -1 {
+				resp.Status = "cancelled"
+			}
+			if tempResp.Orders[0].Status == 1 {
+				resp.Status = "on trading"
+			}
+			if tempResp.Orders[0].Status == 2 {
+				resp.Status = "filled partially"
+			}
+			if tempResp.Orders[0].Status == 3 {
+				resp.Status = "Filled totally"
+			}
+			if tempResp.Orders[0].Status == 4 {
+				resp.Status = "Cancelling"
+			}
+			resp.Price = tempResp.Orders[0].Price
+			resp.Amount = tempResp.Orders[0].Amount
+			resp.OrderDate = time.Unix(tempResp.Orders[0].CreateTime, 9)
+			resp.ExecutedAmount = tempResp.Orders[0].DealAmount
+			resp.RemainingAmount = tempResp.Orders[0].Price - tempResp.Orders[0].DealAmount
+			resp.Fee = 0.001
+			for y := int(0); y < len(getOrdersRequest.Currencies); y++ {
+				if getOrdersRequest.Currencies[y].String() != key {
+					continue
+				}
+				if getOrdersRequest.OrderSide == "ANY" {
+					finalResp = append(finalResp, resp)
+					continue
+				}
+				if strings.EqualFold(getOrdersRequest.OrderSide.ToString(), tempResp.Orders[0].Type) {
+					finalResp = append(finalResp, resp)
+				}
+
+			}
+
+		}
+	}
+	return finalResp, nil
+}
+
+// GetOrderHistory retrieves account order information *
+// Can Limit response to specific order status
+func (l *Lbank) GetOrderHistory(getOrdersRequest *exchange.GetOrdersRequest) ([]exchange.OrderDetail, error) {
+	var resp []exchange.OrderDetail
+	for a := range getOrdersRequest.Currencies {
+		p := exchange.FormatExchangeCurrency(l.Name, getOrdersRequest.Currencies[a])
+		b := int64(1)
+		tempResp, err := l.QueryOrderHistory(p.String(), strconv.FormatInt(b, 10), "200")
+		if err != nil {
+			return resp, err
+		}
+		tempData := tempResp.PageLength
+		for tempData == 200 {
+			tempResp, err = l.QueryOrderHistory(p.String(), strconv.FormatInt(b, 10), "200")
+			if err != nil {
+				return resp, err
+			}
+		}
+	}
+	return resp, nil
+}
+
+// GetFeeByType returns an estimate of fee based on the type of transaction *
+func (l *Lbank) GetFeeByType(feeBuilder *exchange.FeeBuilder) (float64, error) {
```

Comment:
This function is not accurate. Returning 0.001 is the actual fee amount here when [trading fees ](https://lbankinfo.zendesk.com/hc/en-gb/articles/360012072873-Trading-Fees) are `0.001 * amount * price`.
There are also [withdrawal fees](https://lbankinfo.zendesk.com/hc/en-gb/articles/360011969854-Withdrawal-Fees). Withdrawal fees could be calculated using  https://api.lbkex.com/v1/withdrawConfigs.do or a map

This change should be done in LBank and called from the wrapper